tests/Doctrine/Tests/DBAL/Driver/AbstractSQLServerDriverTest.php

```diff
@@ -52,6 +52,8 @@ protected function getDatabasePlatformsForVersions()
             array('11.00.2100.99', 'Doctrine\DBAL\Platforms\SQLServer2012Platform'),
             array('11.00.2101', 'Doctrine\DBAL\Platforms\SQLServer2012Platform'),
             array('12', 'Doctrine\DBAL\Platforms\SQLServer2012Platform'),
+            array('14', 'Doctrine\DBAL\Platforms\SQLServer2014Platform'),
```

Comment:
It tests that database version numbers returned from the server are correctly parsed and mapped to the best suitable platform (auto platform detection)